src/Visual/Renderer.hs

```diff
@@ -44,3 +42,14 @@ positionPicture camera pic = Scale ratio ratio (Translate x y pic)
         offset = unwrap . invertVector . positionToVector . cameraPosition $ camera
         x = fst offset
         y = snd offset
+
+
+
+objectToPicture :: Object -> Picture
+objectToPicture obj = translate' obj $ scale' obj $ appearancePicture . objectAppearance $ obj
+  where scale' = \o -> uncurry Scale $ computeScale (appearanceBox . objectAppearance $ o)
+                                                    (appearanceActualSize . objectAppearance $ o)
+        getOffset o = ( (positionToVector . fst . appearanceBox . objectAppearance $ o) `plus`
```

Comment:
It would be better to use `plus` in prefix form IMO.

Reply:
I'll leave this here just for the record
![capture](https://user-images.githubusercontent.com/36445024/36612620-286a8566-18e8-11e8-9f3e-53643901d209.PNG)

Reply:
done